ql/src/java/org/apache/hadoop/hive/ql/ppd/OpWalkerInfo.java

```diff
@@ -39,11 +43,15 @@
     opToPushdownPredMap;
   private final ParseContext pGraphContext;
   private final List<FilterOperator> candidateFilterOps;
+  private final Map<Operator<?>, Set<ExprNodeColumnDesc>> columnsInPredicates;
```

Comment:
Can we add comments about these data structures and what they will hold?

Reply:
Added comments